### PR TITLE
[Chore] Re-sync moved files

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Re-sync all moved files from SE.

### CODE REVIEW

File: Sources/Notifications/Push Notifications/Notification Types/Content/ZMLocalNotification.swift [hunk 1]
```diff
@@ -24,8 +24,9 @@ import UserNotifications
 ///
 public enum LocalNotificationType {
     case event(LocalNotificationEventType)
+/// TODO Katerina to fix calling state
```
Comment (typfel) [inline Sources/Notifications/Push Notifications/Notification Types/Content/ZMLocalNotification.swift:27]: Can't you restore the `.incomingCall(video: Bool)` logic I added?

File: Sources/Synchronization/Decoding/EventDecoderTest.swift [hunk 2]
```diff
@@ -430,47 +423,5 @@ extension EventDecoderTest {
             XCTAssert(self.eventMOC.saveOrRollback())
         }
     }
-    
-    
-//    fileprivate func payloadForMessage(in conversation: ZMConversation?,
```
Comment (typfel) [inline Sources/Synchronization/Decoding/EventDecoderTest.swift:435]: This should be deleted? Doesn't exist in develop on `wire-ios-sync-engine`.

**Review by typfel — CHANGES_REQUESTED** [review]

File: Sources/Synchronization/Decoding/EventDecoderTest.swift (hunk 2 above)
Comment (KaterinaWire) [reply Sources/Synchronization/Decoding/EventDecoderTest.swift:435]: Yes. I copied `EventDecoderTest` from develop on `wire-ios-sync-engine`.
These 2 methods are in the `MessagingTestBase` extension here, and in the `MessagingTest`  on `wire-ios-sync-engine`

Comment (KaterinaWire) [reply Sources/Synchronization/Decoding/EventDecoderTest.swift:435]: Yes. I copied `EventDecoderTest` from develop on `wire-ios-sync-engine`.
These 2 methods are in the `MessagingTestBase` extension here, and in the `MessagingTest` on `wire-ios-sync-engine`

File: Sources/Notifications/Push Notifications/Notification Types/Content/ZMLocalNotification.swift (hunk 1 above)
Comment (KaterinaWire) [reply Sources/Notifications/Push Notifications/Notification Types/Content/ZMLocalNotification.swift:27]: There is no `.incomingCall(video: Bool)` for `LocalNotificationType` in `wire-ios-sync-engine `now.

**Review by typfel — APPROVED** [review]

File: Sources/Notifications/Push Notifications/Notification Types/LocalNotificationContentType.swift [hunk 3]
```diff
@@ -82,14 +82,15 @@ public enum LocalNotificationContentType: Equatable {
...
-            if let textMessageData = message.textData {
+            if message.ephemeral.hasText {
+                let textMessageData = message.ephemeral.text
                 let quotedMessage = getQuotedMessage(textMessageData, conversation: conversation, in: moc)
                 self = .ephemeral(isMention: textMessageData.isMentioningSelf(selfUser), isReply: textMessageData.isQuotingSelf(quotedMessage))
             } else {
                 self = .ephemeral(isMention: false, isReply: false)
             }
 
-        case .text, .edited:
```
Comment (David-Henner) [inline Sources/Notifications/Push Notifications/Notification Types/LocalNotificationContentType.swift:92]: should the `.edited` case be handled somewhere else?

Comment (KaterinaWire) [reply Sources/Notifications/Push Notifications/Notification Types/LocalNotificationContentType.swift:92]: Looks like no. This was recently changed in SE. The commit comment says:
`Don't generate notifications for edit messages `

Comment (KaterinaWire) [reply Sources/Notifications/Push Notifications/Notification Types/LocalNotificationContentType.swift:92]: Looks like no. This was recently changed in SE. The commit comment says:
`Don't generate notifications for edit messages`

**Review by David-Henner — APPROVED** [review]

**Pull request merged**